index.d.ts

```diff
@@ -0,0 +1,176 @@
+import * as React from 'react';
+import * as PIXI from 'pixi.js';
+
+declare module 'react-pixi-fiber' {
+
+  /**
+   * The set of object keys in T not in U.
+   *
+   * Attribution: https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766:
+   */
+  export type Diff<T extends string, U extends string> = ({ [P in T]: P } & { [P in U]: never } & { [x: string]: never })[T];
+
+  /**
+   * An object with keys in T not in U.
+   *
+   * Attribution: https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766:
+   */
+  export type Omit<T, K extends keyof T> = Pick<T, Diff<keyof T, K>>;
+
+  /** The structure of a `StagedComponent<P, S>` or `StagedStatlessComponent<P>`'s context. */
+  export interface StagedComponentContext {
+    app: PIXI.Application;
+  }
+
+  /** A `React.Component` with context configured and typed. */
+  export abstract class StagedComponent<P = {}, S = {}> extends React.Component<P, S> {
+    context: StagedComponentContext;
+  }
+
+  /** A `React.StatlessComponent<P>` with typed context.  To configure, use the `lift` or `connect` HoC's. */
+  export interface StagedStatlessComponent<P = {}> {
```

Comment:
Typo: `Statless` -> `Stateless`

Reply:
This and `StagedComponent` no longer have a place.  Removing.